pkgs/servers/dns/bind/default.nix

```diff
@@ -51,7 +51,7 @@ stdenv.mkDerivation rec {
     "--with-aes"
   ] ++ lib.optional stdenv.isLinux "--with-libcap=${libcap.dev}"
     ++ lib.optional enableSeccomp "--enable-seccomp"
-    ++ lib.optional enableGSSAPI "--with-gssapi=${libkrb5.dev}"
+    ++ lib.optional enableGSSAPI "--with-gssapi"
```

Comment:
@Luflosi if the error is ` configure: error: --with-gssapi requires yes|no|auto|/path/to/krb5-config`, couldn't you provide the path instead of removing it?

Reply:
When I made this PR I looked at the build log of the two versions and determined that the configure script is properly detecting the dependency as before, so this is fine. Nevertheless I changed it to now specify the dependency explicitly again.